### PR TITLE
Add FAQs page

### LINKED ISSUE #293: Add 'FAQs' page to RTD
In the first instance, this page would take questions that people ask in the slack or anywhere else and give them a central place to exist. 

- [ ] Create page
- [ ] Collect existing questions from slack
- [x] Add questions to page

### PR DESCRIPTION
**Pull request by noahsprent**
Initial FAQs page, just using appropriate questions that have been asked on Slack so far, addresses #293

### CODE REVIEW

**Comment by noahsprent** [comment]
@plouisross Care to review the docs and check that the questions are all ok? You can see the page here: https://igem-distribution--309.org.readthedocs.build/en/309/getting_started/faq.html

File: docs/getting_started/faq.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
+(In)Frequently Asked Questions
+==============================
+
+When working on an issue is it better to create a Branch then Merge changes into the Develop branch or is it better to create a Fork of the repository and then initiate a Pull Request?
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+Either method is acceptable. In general, the Branch method is more suitable for contributors who are already familiar with the Git environment. The Fork method is more appropriate for those who have less Git experience.
```
Comment (plouisross) [inline docs/getting_started/faq.rst]: Start the sentence that begins with "In general" on a new line

File: docs/getting_started/faq.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+How do I run the tests locally? 
+-------------------------------
+
+The tests can be run using Pytest. The script that GitHub actions uses for tests like this can be found `here <https://github.com/iGEM-Engineering/iGEM-distribution/blob/develop/.github/workflows/python-test.yml>`__
+
+I want to update the designation of existing packages in .xslx, noting, for example, whether they are synthetic or not.  I can update the .xls file, but can’t figure out how to update the csv files in views that the README uses to generate itself. How can the scripts be rerun ?
```
Comment (plouisross) [inline docs/getting_started/faq.rst]: add a period (".") before csv.

File: docs/getting_started/faq.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+-------------------------------
+
+The tests can be run using Pytest. The script that GitHub actions uses for tests like this can be found `here <https://github.com/iGEM-Engineering/iGEM-distribution/blob/develop/.github/workflows/python-test.yml>`__
+
+I want to update the designation of existing packages in .xslx, noting, for example, whether they are synthetic or not.  I can update the .xls file, but can’t figure out how to update the csv files in views that the README uses to generate itself. How can the scripts be rerun ? 
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+The required steps are detailed in the “Scripts” README file: 
+
+1. See what is being run on push to build the distribution `here <https://github.com/iGEM-Engineering/iGEM-distribution/blob/develop/.github/workflows/synchronize.yml>`__
```
Comment (plouisross) [inline docs/getting_started/faq.rst]: what does "run on push" mean ??

File: docs/getting_started/faq.rst [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+2. In 'directories_and_excel', there is a section that executes run: ``python3 scripts/export_csvs.py``. Within the code for that script locate ``scriptutils.export_csvs(p)`` that function can be found in ``package_specification.py``.  This is the code that converts the .xlsx file into a csv format that can be exported.
+
+To run the entire process and actually build the distribution from the .xlsx files that is then output in .md summary files that can be accessed locally, run the python scripts in the scripts README sequentially. It is necessary to follow all the steps specified.
```
Comment (plouisross) [inline docs/getting_started/faq.rst]: Propose the following reformulation:  
To run the entire process and build the distribution from the .xlsx files,  run the python scripts in the scripts README sequentially.   The output takes the form of locally-accessible .md summary files

File: docs/getting_started/faq.rst [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+Within the process, the specific steps relevant to the question are:
```
Comment (plouisross) [inline docs/getting_started/faq.rst]: Proposed reformulation
Highlighted below are the specific steps within the process that are most directly relevant to the question:

File: docs/getting_started/faq.rst [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+1. It looks like the .mds are made in the job ``build_packages`` when the python script ``generate_markdown`` is run here.
+
+2. This runs the script ``generate_package_summary`` in ``generate_markdown.py``
+
+**tl;dr**:
+
+1. Setup a python venv with requirements from ``requirements.txt``
```
Comment (plouisross) [inline docs/getting_started/faq.rst]: should "venv" be italicized for clarity ?

**Comment by noahsprent** [comment]
Changes made, build looks good

**Pull request merged**